website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
+page_title: Upgrading to 1.10.0
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrading to 1.10.0
+
+<EnterpriseAlert />
+
+## Introduction
+
+This guide explains how to best upgrade a single Consul Enterprise datacenter to v1.10.0
+from a version of Consul that is forwards compatible with v1.10. If you are on a major 
+version of Consul prior to 1.8, please follow the link for the version you are on from 
+[here](/docs/upgrading/instructions). If you are already on major versions 1.8 or 1.9
+this guide will go over the procedures required for upgrading to v1.10. If you have
+multiple Consul datacenters then they are upgraded in the normal order with each
+upgrade taking into account the instructions below.
+
+For the open source version of Consul please follow the 
+[General Upgrade Process](/docs/upgrading/instructions/general-process).
+
+~> You can only upgrade to Consul Enterprise 1.10 from a version of Consul Enterprise 
+1.8 >= 1.8.13 or 1.9 >= 1.9.7. Other versions of Consul Enterprise are not forwards
+compatible with v1.10 and will cause issues during the upgrade that could result in
+agents failing to start.
+
+## Requirements
+
+- All Consul servers, clients and snapshot agents should be on a version of Consul >= 1.8.0 and < 1.10.0.
+
+## Assumptions
+
+This guides makes the following assumptions:
+
+- You are familiar with the [General Upgrade Process](/docs/upgrading/instructions/general-process).
+- You have the ability to run Consul CLI commands. 
```

Comment:
Do we need to state that they also have an operator token?

Reply:
Maybe a conditional requirement if ACLs are in use?